### PR TITLE
Update new instances

### PR DESCRIPTION
**Pull request by ric9176**
This PR updates the site with the new upcoming advanced and gql courses. As such a new instance page was created for gql and folder structure adapted to allow for multiple london advanced courses. Although these were added in data.js, all instances were added via upmentoring so these changes won't effect upcoming data integration :smile:

### CODE REVIEW

File: src/pages/graphql-bootcamp-london.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+          { to: '/', label: 'Home' },
+          { to: '/graphql-bootcamp-london', label: 'GraphQL bootcamp' },
+        ]}
+      />
+      <Header
+        titleLines={[
+          'React Redux GraphQL Bootcamp',
+          `${training.dates} - London`,
+        ]}
+        subtitle="Take your dev career to the next level by mastering<br />React, Redux, and GraphQL - in just 1 week!"
```
Comment (alexlbr) [inline src/pages/graphql-bootcamp-london.js]: @ric9176 we don't do Redux in the GraphQL bootcamp

File: src/pages/graphql-bootcamp-london.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    <React.Fragment>
+      <Breadcrumb
+        path={[
+          { to: '/', label: 'Home' },
+          { to: '/graphql-bootcamp-london', label: 'GraphQL bootcamp' },
+        ]}
+      />
+      <Header
+        titleLines={[
+          'React Redux GraphQL Bootcamp',
```
Comment (alexlbr) [inline src/pages/graphql-bootcamp-london.js]: @ric9176 shall we remove "React Redux" form the title?

File: src/pages/graphql-bootcamp-london.js [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+                  }}
+                />
+                <InstallmentsCard price={2160} />
+                <Newsletter />
+              </Col>
+              <Col xs={12} md={6} lg={4} lgOffset={1}>
+                <Video youtubeId="2-IPT7Plsfc" />
+                <TrainingDetails
+                  date={training.dates}
+                  timing="Monday-Friday, 9am - 6:30pm"
```
Comment (alexlbr) [inline src/pages/graphql-bootcamp-london.js]: @ric9176 do we do Wed to Tue in the end?

File: src/pages/advanced-react-redux-graphql-bootcamp-london/2.js [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+                  price: training.price,
+                  currency: training.currency,
+                }}
+              />
+            </Col>
+            <Col xs={12} md={6} lg={4} lgOffset={1}>
+              <Video youtubeId="yvROXLQ1jHg" />
+              <TrainingDetails
+                date={training.dates}
+                timing="9:30am - 6:30pm"
```
Comment (alexlbr) [inline src/pages/advanced-react-redux-graphql-bootcamp-london/2.js]: @ric9176 9:30 or 9 am?

File: src/config/data.js [hunk 5]
```diff
@@ -160,10 +138,77 @@ const upcomingBootCamps = [
...
+    currency: 'gbp',
+    image: LONDON_LOCATION,
+  },
 ]
 export const trainings = [
   ...upcomingPartTimes,
   ...upcomingBootCamps,
+  ...upcomingAdvanced,
+  {
+    dates: 'March 31 to April 6, 2019',
```
Comment (alexlbr) [inline src/config/data.js]: @ric9176 are we still using this one? should we remove the data that is in the past? it's a bit messy otherwise

Comment (ric9176) [reply src/config/data.js]: true, I'll delete all past ones and push

**Comment by alexlbr** [comment]
👍

**Pull request merged**